projects/cobbler-frontend/src/app/app-events/app-events.component.html

```diff
@@ -1,25 +1,24 @@
-<app-manage-menu></app-manage-menu>
-
-
-
-<div class="right-column" id="dataScreen">
-  <router-outlet></router-outlet>
-  <div class="AppEvents-div">
-
-    <h1 class="title">EVENTS</h1>
-    <div class="list-group">
-      <ul>
-        <li class="list-group-item">data 01</li>
-        <li class="list-group-item">data 02</li>
-        <li class="list-group-item">data 03</li>
-
-      </ul>
-
+<div id="wrapper" class="d-sm-flex flex-row">
```

Comment:
Is it possible that we are not required to add the wrapper to each component? I mean this is repetitive and error prone.

Reply:
I agree with the repetiveness. It can be added to app.component.html and would apply to all pages including login. Is there any other pages that should not have the sidebar? I can look to see if there’s a way to exclude certain pages if we put it globally.

Reply:
I think apart from the login we can and should use it everywhere...